.github/workflows/test.yml

```diff
@@ -24,3 +24,25 @@ jobs:
         run: cargo install cargo-make
       - name: cargo make - test
         run: cargo make test
+
+  end-to-end:
+    name: end-to-end tests stable on ubuntu-latest
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          override: true
+      - name: Install cargo make
+        run: cargo install cargo-make
+      - name: cargo make - docker-build-node
+        run: cargo make docker-build-node
+      - name: cargo make - docker-run-node
+        run: cargo make docker-run-node
+      - name: cargo install - miden-client
+        run: cargo install miden-client --features concurrent,testing
+      - name: cargo make - test-end-to-end
+        run: cargo make test-end-to-end
```

Comment:
Question: is the docker image built with `concurrent,testing` by default? Should we specify these features explicilty?

Reply:
Indeed the docker image is built with the `concurrent,testing` features enabled by default.

Could you clarify by specify these features explicitly?

Reply:
On line 46 we explicitly build `miden-client` with `concurrent,testing` features. Could we do something similar on line 42?

The main motivation is to make sure that that changing the defaults for docker build does not break this task.